internal/services/logic/logic_app_standard_resource.go

```diff
@@ -519,6 +555,30 @@ func resourceLogicAppStandardUpdate(d *pluginsdk.ResourceData, meta interface{})
 		}
 	}
 
+	if d.HasChange("ftp_publish_basic_authentication_enabled") && !d.IsNewResource() {
```

Comment:
Adding a comment for why we're doing `!d.IsNewResource()` in an update would be good. Especially if we're rewriting these resources in the future